doc/ale-python.txt

```diff
@@ -145,6 +145,14 @@ g:ale_python_isort_executable                   *g:ale_python_isort_executable*
   See |ale-integrations-local-executables|
 
 
+g:ale_python_isort_options                      *g:ale_python_isort_options*
```

Comment:
Travis complained about the alignment of the doctag here. I'll fix it.

Reply:
sorry about that!